lib/async_hooks.js

```diff
@@ -491,3 +492,15 @@ module.exports = {
   emitAfter: emitAfterS,
   emitDestroy: emitDestroyS,
 };
+
+// currentId was renamed to currentAsyncId. This was in 8.1.0 during the
+// experimental stage so the alias can be removed at any time, we are just
+// being nice :)
+Object.defineProperty(module.exports, 'currentId', {
+  configurable: true,
```

Comment:
I think we should hide it, i.e. `{ configurable: false, enumerable: false }` so it will work but won't be discoverable by inspection.